.github/workflows/pr.yaml

```diff
@@ -358,6 +359,16 @@ jobs:
         node_type: "cpu4"
         build_type: pull-request
         run_script: "ci/cudf_pandas_scripts/pandas-tests/diff.sh"
+  narwhals-tests:
```

Comment:
Should these tests specify `continue-on-error: true` or equivalent as this will always be susceptible for breakages on the narwhals main branch?

Reply:
Yes, will do. I think this is a general problem anytime we're testing on the main dev branch of a repo. And since narwhals tests aren't included in the python package (like polars), we have to use the main dev branch I guess.